cypress/integration/centerSpider.spec.js

```diff
@@ -1,17 +1,13 @@
 'use strict';
 
-describe(('Spider application'), () => {
+describe('Spider application', () => {
   before(() => {
-    cy.visit('/');
+    cy.visit('');
   });
 
   it('should center the spider by left border', () => {
-    // NOTE: waiting for transition
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(1000);
-
-    cy.get('.spider').then(($spider) => {
-      cy.get('.wall').then(($wall) => {
+    cy.get('.wall').then(($wall) => {
```

Comment:
try
cy.get('.wall').first().then(($wall)) => ...

this way you probably won't need wall.get(0).

Reply:
tried, it doesn't work this way